src/sass/theme.sass

```diff
@@ -40,11 +27,17 @@
 @import wyrm_core/table
 @import wyrm_core/type
 
-// Theme specific styles. These are likely the files you want to edit.
-@import theme_breadcrumbs
+//-- Theme specific styles. The content below is likely what you want to edit.
+
 @import theme_layout
 @import theme_badge
 @import theme_rst
-@import theme_mathjax
-@import theme_font_awesome_compatibility
-@import theme_font_local
+@import theme_fonts
+
+/* MATHJAX */
+
+span[id*='MathJax-Span']
+  color: $mathjax-color
+
+.math
+  text-align: center
```

Comment:
I think this should be moved to https://github.com/readthedocs/sphinx_rtd_theme/blob/master/src/sass/_theme_rst.sass